open-company/open-company-email
# Email refresh

https://trello.com/c/U6ugzNm0

Transactional Emails:

- Carrot invite
- Verify email
- Reset Password
- Share post
- Invite to private board

Then we have the daily/weekly digest emails.

This requires storage PR: https://github.com/open-company/open-company-storage/pull/118


To test:

- [x] Run with storage branch (see above)
- [x] Run digest server

For each verify you receive the email with no problem and it looks right. Check as many email clients as you have handy (mobile native, desktop native, web,...)

- [x] Carrot invite: Invite a user by email to your team
- [x] Verify email: register as a new user by email (uninvited)
- [x] Reset password: click the reset password link in the login dialog for an email user
- [x] Share post: share a post to an email address
- [x] Private board invite: Have an email user in your org, and add them to a private board
- [x] Daily digest: force a daily digest with http://localhost:3008/_/email/weekly Click through to the post(s)
- [x] Weekly digest: force a weekly digest with http://localhost:3008/_/email/weekly Click through to the post(s)
- [ ] Merge branches
- [ ] Deploy branches
- [ ] Refactor Carrot UI
